### ISSUE TITLE
Investigate: StarterMvc.Clr.DotnetPublish gets stuck

### ISSUE THREAD

**Issue by DavidObando** [aspnet/Performance#40]
When executing .\build.cmd I'm seeing, consistently, that dnx.exe lowers its CPU load (however, never to less than 3% CPU) and gets stuck there indefinitely until killed, resulting in a test failure. This looks like some sort of lock.

I attached a debugger and saw a number of threads waiting (including one waiting on a System.Data call). I haven't figured out the source of the issue yet.

I suggest we run this test under a debugger and investigate. A simple repro:
```
λ pushd test\Microsoft.AspNetCore.Tests.Performance
λ dnu restore
λ dnu build
λ dnx test
```

**Comment by DavidObando** [comment]
This may be a dnx issue. De-prioritize but don't close. Verify if the issue persists after our move to CLI.

**Comment by DavidObando** [comment]
Fixed with aspnet/Performance@53682b2.
